device_coupler/ovs_helper.py

```diff
@@ -17,26 +17,39 @@ def __init__(self):
         self._run_shell = partial(ShellCommandHelper().run_cmd, capture=True)
         self._run_shell_no_raise = partial(ShellCommandHelper().run_cmd, capture=True, strict=False)
 
-    def create_vxlan_endpoint(self, port, remote_ip, vni, local_ip=None):
+    def _set_interface_up(self, interface, no_raise=False):
+        if no_raise:
```

Comment:
This can be simplified to 
`fn = self. _run_shell_no_raise if no_raise else self. _run_shell`
`fn('sudo ip link set %s up' % interface)`

Reply:
Also no_raise with default False can also be simplified to  `raise=True`

Reply:
Done.